Agents/IfcOwlConverterAgent/IfcOwlConverter/pom.xml

```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>uk.ac.cam.cares.jps</groupId>
+    <artifactId>ifcowlconverter</artifactId>
+    <version>1.0.0</version>
+    <packaging>war</packaging>
+
+    <!--  Project properties  -->
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.release>11</maven.compiler.release>
+        <jps.base.version>1.21.1</jps.base.version>
```

Comment:
I guess something more recent is not possible due to dependency incompatibility?

Reply:
Could I clarify if you mean for the JPS-base lib dependency? If so, updating the jps base lib to the latest version is possible. The dependency incompatibility is only for some Apache Jena packages. This have been resolved by explicitly stating the required 4.1.0 version within this project pom file.

Reply:
If it doesn't break your code, please use the most recent base lib version possible.

Reply:
Alright, I have updated the base lib version, and tested that it works.